model/project_parser.go

```diff
@@ -84,13 +90,10 @@ type ParserProject struct {
 	ExecTimeoutSecs    *int                       `yaml:"exec_timeout_secs,omitempty" bson:"exec_timeout_secs,omitempty"`
 	Loggers            *LoggerConfig              `yaml:"loggers,omitempty" bson:"loggers,omitempty"`
 	CreateTime         time.Time                  `yaml:"create_time,omitempty" bson:"create_time,omitempty"`
-	// List of yamls to merge
-	Include []Include `yaml:"include,omitempty" bson:"include,omitempty"`
 
 	// Matrix code
 	Axes []matrixAxis `yaml:"axes,omitempty" bson:"axes,omitempty"`
-} // End of ParserProject struct
-// Comment above is used by the linter to detect the end of the struct.
+} // End of ParserProject mergeable fields (used by the linter).
```

Comment:
I think this comment is less understandable to me now. Can we maybe change it to (this comment is used by the linter)?